src/v2/Components/FullBleedHeader/FullBleedHeaderFader.tsx

```diff
@@ -0,0 +1,72 @@
+import { Box } from "@artsy/palette"
+import { FC, useEffect, useRef } from "react"
+import styled from "styled-components"
+import { useCursor } from "use-cursor"
+import { useMode } from "v2/Utils/Hooks/useMode"
+import { wait } from "v2/Utils/wait"
+import { FullBleedHeader, FullBleedHeaderProps } from "./FullBleedHeader"
+
+interface FullBleedHeaderFaderProps {
+  figures: FullBleedHeaderProps[]
+}
+
+export const FullBleedHeaderFader: FC<FullBleedHeaderFaderProps> = ({
+  figures,
+  children,
+}) => {
+  const [mode, setMode] = useMode<"Resting" | "Transitioning">("Resting")
+
+  const { index, cursor, handleNext } = useCursor({ max: figures.length })
+
+  const currentFigure = figures[index]
+  const nextFigure = figures[(cursor + 1) % figures.length]
+
+  const ref = useRef<HTMLDivElement | null>(null)
+
+  useEffect(() => {
+    const init = async () => {
+      await wait(5000)
+      setMode("Transitioning")
+      await wait(1500)
+      setMode("Resting")
+      handleNext()
+
+      init()
+    }
+
+    init()
+  }, [handleNext, setMode])
+
+  return (
+    <Box ref={ref as any} position="relative">
+      {[currentFigure, nextFigure].map((figure, i) => (
+        <FullBleedHeaderWithTransition
+          relativeTo={ref}
+          key={figure.src}
+          {...figure}
+          {...(i === 0
+            ? {}
+            : {
+                position: "absolute",
+                top: 0,
+                style:
+                  mode === "Transitioning"
+                    ? {
+                        opacity: 1,
+                      }
+                    : {
+                        opacity: 0,
+                        pointerEvents: "none",
+                      },
+              })}
+        >
+          {figure.children ?? children}
+        </FullBleedHeaderWithTransition>
+      ))}
+    </Box>
```

Comment:
Stacks the current and next header on top of one another. Waits a moment then transitions the opacity of the next one which is stacked on top. Once it's done transitioning we move to the next, moving the top to the bottom, then stacking a new one on top with an opacity of 0. Then repeat.